app/src/main/kotlin/com/hippo/ehviewer/EhApplication.kt

```diff
@@ -177,7 +179,9 @@ class EhApplication :
             if (isAtLeastO) {
                 add(HardwareBitmapInterceptor)
             }
+            add(QRCodeInterceptor)
             add(CropBorderInterceptor)
```

Comment:
I think we don't need to crop border if the image is ad, so maybe swap the order?

Reply:
Done